src/SFA.DAS.EmployerAccounts.Web/Helpers/HomepagePanelViewHelper.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Linq;
+using SFA.DAS.EmployerAccounts.Web.ViewModels;
+
+namespace SFA.DAS.EmployerAccounts.Web.Helpers
+{
+    public class NextActionPanelViewHelper : INextActionPanelViewHelper
```

Comment:
This is too specific to PAYE.  I think the panel name should be passed in so it can be used elsewhere.  Also rather than return a AccountDashboardViewModel return a NextView (?} object or ViewInfo object with a name property that has the name of the view to render next.  That way this helpers concern is just to give information about the next view to the caller rather than tying it to a specific viewmodel.

Reply:
I agree, however I think once this is merged with 454 there will be a large refactor here anyway so this code will change anyway